modules/standard/Subprocess.chpl

```diff
@@ -447,12 +477,17 @@ module Subprocess {
     var pid:int;
     var err:syserr;
 
-    if isIntegralType(stdin.type) then stdin_fd = stdin;
-    else compilerError("only FORWARD/CLOSE/PIPE/STDOUT supported");
-    if isIntegralType(stdout.type) then stdout_fd = stdout;
-    else compilerError("only FORWARD/CLOSE/PIPE/STDOUT supported");
-    if isIntegralType(stderr.type) then stderr_fd = stderr;
-    else compilerError("only FORWARD/CLOSE/PIPE/STDOUT supported");
+    if stdin.type == pipeStyle || isIntegralType(stdin.type) then
+      stdin_fd = pipeStyleToInt(stdin);
+    else compilerError("only pipeStyle.forward/close/pipe/stdout supported");
```

Comment:
At a glance, this seems like it lets through cases I wouldn't expect.  But that's not really something you've introduced.  If you have the bandwidth, I'd be curious if there are tests that lock in providing numbers we don't expect

Reply:
My guess was that it's letting through other integers so it can handle any file descriptor, not just the ones we've defined `pipeStyle`s for. I haven't gone digging for tests though.